<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountInfoWidget</class>
 <widget class="QWidget" name="AccountInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account Info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="m_identificationLayout" stretch="9,10">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="m_credentialsLayout">
       <item>
        <widget class="QGroupBox" name="m_photoIdGroup">
         <property name="title">
          <string>Photo ID</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_20">
          <item>
           <widget class="QGraphicsView" name="m_photoIdView"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="m_photoButtonLayout">
            <item>
             <spacer name="m_photoButtonSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="m_changePhotoButton">
              <property name="text">
               <string>Change Photo</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_passwordGroup">
         <property name="title">
          <string>Set Password</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_21">
          <item>
           <layout class="QGridLayout" name="m_passwordLayout" columnstretch="0,0,0" columnminimumwidth="0,0,0">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item row="0" column="1">
             <widget class="QLabel" name="m_confirmPasswordLabel">
              <property name="text">
               <string>Confirm Password</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="m_newPasswordInput">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="m_confirmPasswordInput">
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="m_updatePasswordButton">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="m_newPasswordLabel">
              <property name="text">
               <string>New Password</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="m_identificationGroupBox">
       <property name="title">
        <string>Identification</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="m_firstNameLabel">
          <property name="text">
           <string>First Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="m_firstNameInput"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="m_lastNameLabel">
          <property name="text">
           <string>Last Name:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="m_lastNameInput"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="m_registeredLabel">
          <property name="text">
           <string>Registered:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="m_registeredInput">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="m_lastLoginLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Last Login:</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="m_lastLoginInput">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="m_accountIdLabel">
          <property name="text">
           <string>Account ID:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="m_accountIdInput">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="m_emailLabel">
          <property name="text">
           <string>E-mail:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="m_emailInput"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="m_addressLineOneLabel">
          <property name="text">
           <string>Address Line 1:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="m_addressLineOneInput"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="m_addressLineTwoLabel">
          <property name="text">
           <string>Address Line 2:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="m_addressLineTwoInput"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="m_addressLineThreeLabel">
          <property name="text">
           <string>Address Line 3:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="m_addressLineThreeInput"/>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="m_cityLabel">
          <property name="text">
           <string>City:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="m_cityInput"/>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="m_provinceLabel">
          <property name="text">
           <string>Province/State:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLineEdit" name="m_provinceInput"/>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="m_countryLabel">
          <property name="text">
           <string>Country:</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QComboBox" name="m_countryInput"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="m_userNotesLayout">
     <property name="title">
      <string>User Notes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="m_userNotesInput">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_firstNameInput</tabstop>
  <tabstop>m_lastNameInput</tabstop>
  <tabstop>m_registeredInput</tabstop>
  <tabstop>m_lastLoginInput</tabstop>
  <tabstop>m_accountIdInput</tabstop>
  <tabstop>m_emailInput</tabstop>
  <tabstop>m_addressLineOneInput</tabstop>
  <tabstop>m_addressLineTwoInput</tabstop>
  <tabstop>m_addressLineThreeInput</tabstop>
  <tabstop>m_cityInput</tabstop>
  <tabstop>m_provinceInput</tabstop>
  <tabstop>m_countryInput</tabstop>
  <tabstop>m_userNotesInput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
